### DISCUSSION TITLE
V2.0.0

### DISCUSSION THREAD

**Discussion by bpatrik** [bpatrik/pigallery2#742] [General]
# UI/UX update
Main changes:
  * dark mode
  * sticky top bar
  * in line blog support
  * new icons (used with buttons)
  * new icon / page logo
  * top pick sending (sends mail about photos take on the same week but previous years)
  * improving sharing UI
  * refactored settings
  * data patter search ( search query like: `last-7-days:every-year`)

## breaking changes
 * you will need to reset the DB
 * you will need to recreate the config file

## What's Changed
* The spaces for ffmpeg custom options are strict. by @lijunle in https://github.com/bpatrik/pigallery2/pull/597
* Redirect ffmpeg output to logger. by @lijunle in https://github.com/bpatrik/pigallery2/pull/598
* document issue #177 about overriding default sort order by @kagahd in https://github.com/bpatrik/pigallery2/pull/620
* Add matchType to getPositionSearchQuery by @SmartFinn in https://github.com/bpatrik/pigallery2/pull/635
* docs: fix text-align in list by @alexislefebvre in https://github.com/bpatrik/pigallery2/pull/664
* Add SIGTERM handler by @Rickkwa in https://github.com/bpatrik/pigallery2/pull/663
* docs: fix the “fork me” ribbon by @alexislefebvre in https://github.com/bpatrik/pigallery2/pull/665
* docs: add note about apache proxy and sub folders by @alexislefebvre in https://github.com/bpatrik/pigallery2/pull/668
* Top pick sending per e-mail by @bpatrik in https://github.com/bpatrik/pigallery2/pull/685
* Improving e-mail sending #683 by @bpatrik in https://github.com/bpatrik/pigallery2/pull/686
* Showing sharing history and preventing accidental sharing by @bpatrik in https://github.com/bpatrik/pigallery2/pull/693
* New icon #587 by @bpatrik in https://github.com/bpatrik/pigallery2/pull/699
* Replacing app icon with new outline camera + raspberry head icon by @bpatrik in https://github.com/bpatrik/pigallery2/pull/702
* Photo groupping #706 by @bpatrik in https://github.com/bpatrik/pigallery2/pull/707
* Inline or groupped blog #711 by @bpatrik in https://github.com/bpatrik/pigallery2/pull/712
* Matroska container header meta data parsing by @GenericGuy in https://github.com/bpatrik/pigallery2/pull/710
* Adding E2E testing with cypress by @bpatrik in https://github.com/bpatrik/pigallery2/pull/715
* Update docker-compose.yml to refer to version 3.9 rather than 3 by @shankargopal in https://github.com/bpatrik/pigallery2/pull/724
* Update README.md - Typo by @AndrewBudziszek in https://github.com/bpatrik/pigallery2/pull/727
* Update Dutch translation by @mrtnbr in https://github.com/bpatrik/pigallery2/pull/737
* Some small fixes in the Dutch translation. by @mrtnbr in https://github.com/bpatrik/pigallery2/pull/739
* Fixed some text by @mrtnbr in https://github.com/bpatrik/pigallery2/pull/741

## New Contributors
* @SmartFinn made their first contribution in https://github.com/bpatrik/pigallery2/pull/635
* @alexislefebvre made their first contribution in https://github.com/bpatrik/pigallery2/pull/664
* @Rickkwa made their first contribution in https://github.com/bpatrik/pigallery2/pull/663
* @GenericGuy made their first contribution in https://github.com/bpatrik/pigallery2/pull/710
* @shankargopal made their first contribution in https://github.com/bpatrik/pigallery2/pull/724
* @AndrewBudziszek made their first contribution in https://github.com/bpatrik/pigallery2/pull/727

**Full Changelog**: https://github.com/bpatrik/pigallery2/compare/1.9.5...2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/bpatrik/pigallery2/releases/tag/2.0.0'>V2.0.0</a>.</em>

**Comment by TopoR** [comment]
That's an awesome update!
Huge thanks for the dark theme and ability to easily change preview size!

**Comment by wechsler42** [comment]
Thanks a lot Patrik for releasing version 2 with all the awesome new features 👍

**Comment by lythandas** [comment]
Thanks for everything guys, this v2 is a great step forward!
